README.md

```diff
@@ -18,6 +18,8 @@ var client = new RealtimeClient(process.env.REALTIME_URL)
 client.connect()
 ```
 
+REALTIME_URL is `'ws://localhost:4000/socket'` when developing locally and `'wss://aaaaabbbbbcccccddddd.supabase.co/realtime/v1'` (remember to replace the project ref) when connecting to your Supabase project.
```

Comment:
should we change `aaaaabbbbbcccccddddd` to `<project_ref>` or something similar?

Reply:
@kiwicopple sure thing, i was hesitant to do so at first in case new users didn't know what a `project ref` was but will make the change